Tests/AtomsTests/Core/StoreContextTests.swift

```diff
@@ -95,9 +95,9 @@ final class StoreContextTests: XCTestCase {
         XCTAssertEqual(store.graph.dependencies, [key: [dependency0Key]])
         XCTAssertEqual(store.graph.children, [dependency0Key: [key]])
         XCTAssertNil(store.state.atomStates[dependency1Key])
-        XCTAssertEqual(observer.assignedAtomKeys, [dependency0Key])
-        XCTAssertEqual(observer.changedAtomKeys, [dependency0Key])
-        XCTAssertTrue(observer.unassignedAtomKeys.isEmpty)
+        XCTAssertEqual(observer.assignedAtomKeys, [dependency0Key, dependency1Key])
+        XCTAssertEqual(observer.changedAtomKeys, [dependency0Key, dependency1Key])
+        XCTAssertEqual(observer.unassignedAtomKeys, [dependency1Key])
```

Comment:
Terminated transaction now falls back to `read` and it registers and releases immediately.